www/src/public/js/controllers/builder/main.js

```diff
@@ -1912,7 +1963,7 @@
             var total = fromBaseStats + fromKSMStats + fromStatQuests + fromItems + fromSpells + fromBonus;
 
             // apply total bonuses
-            total += getStatTotalBonus(statName, total);
+            // total += getStatTotalBonus(statName, total);
```

Comment:
I'm guessing this was a mistake?

Reply:
oh I see why you did this, I would prefer if you removed the "dam" case in the function, rather than commenting out the function completely

Reply:
Corrected.  I decided to comment out the "dam" case instead of deletion like we talked about.  I figured it hurts nothing and leaves a bit of a cookie crumb for someone coming along later to add something if needed as a use case.